.editorconfig

```diff
@@ -1,27 +1,39 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+# WordPress Coding Standards
+# https://make.wordpress.org/core/handbook/coding-standards/
+
+root = true
+
 [*]
-charset=utf-8
-end_of_line=lf
-insert_final_newline=false
-indent_style=space
-indent_size=4
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
 
-[{.babelrc,.stylelintrc,.eslintrc,jest.config,*.json,*.jsb3,*.jsb2,*.bowerrc}]
-indent_style=space
-indent_size=2
+[*.yml]
+indent_style = space
+indent_size = 2
 
-[*.js.map]
-indent_style=space
-indent_size=2
+[*.md]
+trim_trailing_whitespace = false
 
-[*.scss]
-indent_style=space
-indent_size=2
+[{*.txt,wp-config-sample.php}]
+end_of_line = crlf
+
+# ====================================================================
+# The rules above ↑this↑ line are the exact contents of WordPress' config file.'
+# Github mirror : https://github.com/WordPress/wordpress-develop/blob/trunk/.editorconfig
+# SVN original  : https://core.trac.wordpress.org/browser/trunk/.editorconfig
 
-[{*.yml,*.yaml}]
-indent_style=space
-indent_size=2
+# ======= Custom Yard rules =======
+[*.{js},*.{jsx}, *.{ts}, *.{tsx}]
```

Comment:
Ik zou voor een standaard gaan denk ik, dus overal spaces als het hierboven ook zo bepaalt is. Wellicht dat de eigen regels dan weg kunnen?

Reply:
Check, ik doe de eigen regels weg